packages/react-native-reanimated/metro-config/index.js

```diff
@@ -0,0 +1,35 @@
+const COLLAPSED_STACK_REGEX = new RegExp(
+  [
+    // For internal usage in the example app
+    "/packages/react-native-reanimated/.+\\.(t|j)sx?$",
+    // When reanimated is installed as a dependency (node_modules)
+    "/node_modules/react-native-reanimated/.+\\.(t|j)sx?$",
```

Comment:
Does it work with hoisted node_modules? Also, what happens if it's actually the code in Reanimated which crashes?

Reply:
It should work the same as with hoisted `react-native` module where ignore paths are specified in the default metro config. The path will always end width `node_modules/react-native-reanimated`, even if the module is hoisted, so the regex will match a path in such a case as well.

Recordings in the description are made in a project which uses hoisting and the applied metro config worked fine.